yarn.lock

```diff
@@ -6269,7 +6269,7 @@ enhanced-resolve@^4.5.0:
     memory-fs "^0.5.0"
     tapable "^1.0.0"
 
-enhanced-resolve@^5.10.0:
+enhanced-resolve@^5.0.0, enhanced-resolve@^5.10.0:
```

Comment:
this version is higher than the one in master?

Reply:
I think that's just an optimization created by `yarn` when I ran `yarn install`. The actual version used remains the same:
https://github.com/gravitational/webapps/blob/master/yarn.lock#L6289-L6294

Reply:
weird because in this PR (also a backport) matches one in master: https://github.com/gravitational/webapps/pull/1136/files#diff-51e4f558fae534656963876761c95b83b6ef5da5103c4adef6768219ed76c2deR6288

Reply:
I'm not sure what might actually cause this to jump around. Maybe `yarn` versions? I'm on `1.22.19`.

cc @ryanclark

Reply:
I think it'll be okay. When both PRs are merged, I imagine we'd see something like 

```
enhanced-resolve@^5.0.0, enhanced-resolve@^5.9.3, enhanced-resolve@^5.10.0:
```

Which will all resolve to `enhanced-resolve@5.10.0`